apps/bare-expo/react-native.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  dependencies: {
+    ...require('expo-dev-client/dependencies'),
```

Comment:
I like this solution, however, I just noticed one drawback. How do we remove those dependencies in release builds?

Reply:
On iOS I'm doing this: https://github.com/expo/expo/blob/2a7fa3a71d899a935e48f7468102685c06f8a181/packages/expo-dev-client/expo-dev-client.podspec#L17-L19

Maybe that's enough to remove it from the release build? I wonder if we can do something similar for Android.

Reply:
On Android, those packages will be removed automatically - you don't need to do anything else. 

However, your solution doesn't work for iOS right now. Packages are included in the project by autlinking. Yes, you're excluding them from the pod, but it doesn't matter cause they're still in the project.